docs/dev/run_the_etl.rst

```diff
@@ -115,23 +262,10 @@ The ``pudl_etl`` script CEMS data does not allow you to select which tables to i
 There are a few notable dependencies to be wary of when fiddling with these
 settings:
 
-- EPA CEMS cannot be loaded without EIA data unless you have existing PUDL database
-  containing EIA. This is because CEMS relies on plant IDs from EIA860.
-
-- EIA Forms 860 and 923 are very tightly related. You can load only EIA 860, but the
-  settings verification will automatically add in a few 923 tables that are needed
-  to generate the complete list of plants and generators. The settings verification
-  will also automatically add all 860 tables if only 923 is specified. This is
-  because of the harvesting process that standardizes duplicate and deviant data
-  between the two sources.
-
-.. warning::
+- The ``ferc_to_sqlite`` job must be executed prior to running ``pudl_etl``
+  job.
 
-    If you are processing the EIA 860/923 data, we **strongly recommend**
-    including the same years in both datasets. We only test two combinations of
-    inputs, as specified by the ``etl_fast.yml`` and ``etl_full.yml`` settings
-    distributed with the package.  Other combinations of years may yield
-    unexpected results.
```

Comment:
These dependences have been removed because all datasets are loaded for every ETL run now.